modules/gdscript/gdscript.cpp

```diff
@@ -2432,27 +2430,20 @@ void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
 		p_words->push_back(*w);
 		w++;
 	}
-
-	List<StringName> functions;
-	GDScriptUtilityFunctions::get_function_list(&functions);
-
-	for (const StringName &E : functions) {
-		p_words->push_back(String(E));
-	}
 }
 
 bool GDScriptLanguage::is_control_flow_keyword(String p_keyword) const {
-	return p_keyword == "break" ||
-			p_keyword == "continue" ||
+	return p_keyword == "if" ||
```

Comment:
Why this was changed? There was no need to reorder things here, and it creates no benefit, again, no need to change this PR, but there was also no need to change anything here.